src/components/Home.vue

```diff
@@ -88,38 +85,51 @@ export default {
     this.loaded = true
   },
   computed: {
-      needs(){
-        return this.$store.state.needs
-      },
-      userName(){
-        if (this.$store.state.users[0]) {
-          return this.$store.state.users[0].firstName
-        }
-        else {
-          return ''
-        }
-      },
-      userStatus(){
-        if (this.$store.state.users[0]) {
-          return this.$store.state.users[0].status
-        }
-        else {
-          return ''
-        }
-      },
-      entries(){
-        if (this.$store.state.entries) {
-          let weekdays = this.$store.state.entries[0].sleep.map(a => a.weekday);
-          let hrsSlept = this.$store.state.entries[0].sleep.map(a => a.hrsSlept);
-          console.log('entries', this.$store.state.entries)
-          //console.log('weekdays', hrsSlept)
-          return this.$store.state.entries
-        }
-        else {
-          return ''
+    needs(){
+      return this.$store.state.needs
+    },
+    userName(){
+      if (this.$store.state.users[0]) {
+        return this.$store.state.users[0].firstName
+      }
+      else {
+        return ''
+      }
+    },
+    userStatus(){
+      if (this.$store.state.users[0]) {
+        return this.$store.state.users[0].status
+      }
+      else {
+        return ''
+      }
+    },
+    sleepEntries(){
+      if (this.$store.state.entries) {
+        return {
+          labels: this.$store.state.weekdays,
+          datasets: [
+            {
+              label: 'Hrs slept',
+              backgroundColor: '#f87979',
+              data: this.$store.state.weeklyEntriesSleep
+            }
+          ]
         }
       }
-
+    },
+    entries(){
+      if (this.$store.state.entries) {
+        let weekdays = this.$store.state.entries[0].sleep.map(a => a.weekday);
+        let hrsSlept = this.$store.state.entries[0].sleep.map(a => a.hrsSlept);
+        console.log('weekdays', this.$store.state.weeklyEntriesSleep)
+        
+        return this.$store.state.entries
+      }
+      else {
+        return ''
```

Comment:
semicolon

You can also change this block to:
`if (!this.$store.state.entries) {
  return '';
}
let weekdays = this.$store.state.entries[0].sleep.map(a => a.weekday);
let hrsSlept = this.$store.state.entries[0].sleep.map(a => a.hrsSlept);
console.log('weekdays', this.$store.state.weeklyEntriesSleep)
        
return this.$store.state.entries;`